tests/model_serving/model_server/maas_billing/utils.py

```
@@ -104,3 +118,130 @@ def llmis_name(client, namespace: str = "llm", label_selector: str | None = None
+    def _attempt_login() -> bool:
+        nonlocal last_exc
+        try:
+            ok = login_with_user_password(
+                api_address=api,
+                user=user,
+                password=password,
+            )
+            if not ok:
+                LOGGER.warning("MaaS RBAC: login returned False for %s; will retry", user)
+                return False
+            return True
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)